meta/main.yml

```diff
@@ -1,68 +1,20 @@
-# SPDX-License-Identifier: MIT
----
 galaxy_info:
-  # Replace with role's author name:
-  author: John Doe <jdoe@corp.com>
-  # Replace with the real description of what is role's purpose:
-  description: Basic template for Linux system roles
-  # Replace with the company the role's author is member of:
-  company: John Doe, Inc.
-
-  # If the issue tracker for your role is not on github, uncomment the next
-  # line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
+  author: Sergei Petrosian <spetrosi@redhat.com>
+  description: Install and configure MSSQL
+  company: Red Hat, Inc.
   license: MIT
-
   min_ansible_version: 2.7
```

Comment:
Since this is a new role, we could start with 2.8 (or even 2.9)?

Reply:
Let's to 2.8 to keep it consistent with the other roles

Reply:
I see that the CI does not test on ansible 2.7 so I have no idea if it works just the same. With 2.8 and 2.9, I had cases where I needed to change the syntax to work on both because it failed on one version and succeeded on another. So I see this 2.7 version here as a risk. This string comes from the template and I just didn't change it. Why be consistent when it is risky @richm?

Reply:
> I see that the CI does not test on ansible 2.7 so I have no idea if it works just the same. With 2.8 and 2.9, I had cases where I needed to change the syntax to work on both because it failed on one version and succeeded on another. So I see this 2.7 version here as a risk. This string comes from the template and I just didn't change it.

It has recently been changed in the template: https://github.com/linux-system-roles/template/blob/master/meta/main.yml#L24

> Why be consistent when it is risky @richm?

I don't think being consistent with the other roles (supporting ansible 2.8 and 2.9) is risky.  We have CI coverage for both 2.8 and 2.9 so any problems that arise should be found and fixed quickly.  We haven't run into a problem with the mssql role that can only be solved by using jinja 2.11 or ansible 2.9.

Reply:
I am saying that supporting the older Ansible 2.7 is risky. I read your message mistakenly, will now change to 2.8.